cloudaux/orchestration/aws/glacier.py

```diff
@@ -0,0 +1,52 @@
+from cloudaux.aws.glacier import describe_vault, get_vault_access_policy, list_tags_for_vault
+from cloudaux.decorators import modify_output
+from flagpole import FlagRegistry, Flags
+
+registry = FlagRegistry()
+FLAGS = Flags('DESCRIBE', 'POLICY', 'TAGS')
+
+
+@registry.register(flag=FLAGS.DESCRIBE)
+def _describe_vault(vault_name, **conn):
```

Comment:
Rename the functions to remove leading `_`.

Reply:
On purpose because these won't be called directly, they'll be called with get_vault and the appropriate flags.

Reply:
👍